config/initializers/active_model_serializers.rb

```diff
@@ -0,0 +1,166 @@
+module AMS
+  module V09
+    class Serializer < ActiveModel::Serializer
+
+       def serializable_hash(adapter_options = nil,
+                             options = {},
+                             adapter_instance = self.class.serialization_adapter_instance)
+         object.nil? ? nil : super
+       end
+    end
+
+    class ArraySerializer < ActiveModel::Serializer::CollectionSerializer
+    end
+
+    class ResourceIdentifier < ActiveModelSerializers::Adapter::JsonApi::ResourceIdentifier
+
+      def initialize(serializer, options)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.